saltstack/base/salt/ccmv2-inverting-proxy-agent/cdp/bin/ccmv2/update-inverting-proxy-agent-values.sh

```diff
@@ -36,22 +36,23 @@ AGENT_CERT_PATH=$5
 TRUSTED_BACKEND_CERT_PATH=$6
 TRUSTED_PROXY_CERT_PATH=$7
 
-VALUES_FILE=/cdp/bin/ccmv2/inverting-proxy-agent-values-${BACKEND_ID}.sh
-
-cat > ${VALUES_FILE} <<EOF
-INVERTING_PROXY_URL=${INVERTING_PROXY_URL}
-BACKEND_ID=${BACKEND_ID}
-BACKEND_ADDRESS=${BACKEND_HOST}:${BACKEND_PORT}
-AGENT_KEY_PATH=${AGENT_KEY_PATH}
-AGENT_CERT_PATH=${AGENT_CERT_PATH}
-TRUSTED_BACKEND_CERT_PATH=${TRUSTED_BACKEND_CERT_PATH}
-TRUSTED_PROXY_CERT_PATH=${TRUSTED_PROXY_CERT_PATH}
-SCHEME=${SCHEME}
-HTTP_PROXY_URL=${HTTP_PROXY_URL}
+CONFIG_FILE=/cdp/bin/ccmv2/config-${BACKEND_ID}.toml
```

Comment:
Now that we know that we would only have a single agent running on a system, can we not suffix with backend id? `config.toml` should be enough for this.

Reply:
I would also recommend updating the service name to remove the backend id. (Not necessarily in the same PR) This would make debugging easier.

Reply:
Ok. I thought about this but decided against in order to keep things in sync with how it is right now. Will make the changes.